package.json

```diff
@@ -7,7 +7,7 @@
   "homepage": "https://nebenan.de/",
   "repository": "good-hood-gmbh/nebenan-ui-kit",
   "bugs": "https://github.com/good-hood-gmbh/nebenan-ui-kit/issues",
-  "version": "5.2.0",
+  "version": "5.3.0",
```

Comment:
Chose to go with a minor bump because apps have to update to sass 1.34 but it wasn't really worth a major bump